src/apify_helpers.js

```diff
@@ -64,11 +79,172 @@ const enrichTaskRun = async (z, run, storeKeysToInclude = []) => {
+
+    // Attach Apify app URL to detail of run
+    run.detailsPageUrl = run.actorTaskId
+        ? `https://my.apify.com/tasks/${run.actorTaskId}#/runs/${run.id}`
+        : `https://my.apify.com/actors/${run.actId}#/runs/${run.id}`;
+
+    // Omit fields, which are useless for Zapier users.
+    return _.omit(run, OMIT_ACTOR_RUN_FIELDS);
+};
+
+// Process to subscribe to Apify webhook
+const subscribeWebkook = async (z, bundle, condition) => {
+    const webhookOpts = {
+        eventTypes: Object.values(WEBHOOK_EVENT_TYPES),
+        condition,
+        requestUrl: bundle.targetUrl,
+    };
+    const response = await wrapRequestWithRetries(z.request, {
+        url: APIFY_API_ENDPOINTS.webhooks,
+        method: 'POST',
+        json: webhookOpts,
+    });
+
+    return response.json;
+};
 
-    return run;
+// Process to unsubscribe to Apify webhook
+const unsubscribeWebhook = async (z, bundle) => {
+    // bundle.subscribeData contains the parsed response JSON from the subscribe
+    const webhookId = bundle.subscribeData.id;
+
+    await wrapRequestWithRetries(z.request, {
+        url: `${APIFY_API_ENDPOINTS.webhooks}/${webhookId}`,
+        method: 'DELETE',
+    });
+
+    return {};
+};
+
+// Gets actor run from bundle clean request and enriches it.
+const getActorRun = async (z, bundle) => {
+    const run = bundle.cleanedRequest.resource;
+    const enrichRun = await enrichActorRun(z, run);
+    return [enrichRun];
+};
+
+/**
+ * Get store by ID or by name
+ * @param z
+ * @param storeIdOrName - Key-value store ID or name
+ */
+const getOrCreateKeyValueStore = async (z, storeIdOrName) => {
+    let store;
+    // The first try to get store by ID.
+    try {
+        const storeResponse = await wrapRequestWithRetries(z.request, {
+            url: `${APIFY_API_ENDPOINTS.keyValueStores}/${storeIdOrName}`,
+            method: 'GET',
+        });
+        store = storeResponse.json;
+    } catch (err) {
+        if (!err.message.includes('not found')) throw err;
+    }
+
+    // The second creates store with name, in case storeId not found.
+    if (!store) {
+        const storeResponse = await wrapRequestWithRetries(z.request, {
+            url: `${APIFY_API_ENDPOINTS.keyValueStores}`,
+            method: 'POST',
+            params: {
+                name: storeIdOrName,
+            },
+        });
+        store = storeResponse.json;
+    }
+    return store;
+};
+
+/**
+ * This method loads additional input fields regarding actor default values.
+ */
+const getActorAdditionalFields = async (z, bundle) => {
+    const { actorId } = bundle.inputData;
+    if (!actorId) return [];
+
+    const actorResponse = await wrapRequestWithRetries(z.request, {
+        url: `${APIFY_API_ENDPOINTS.actors}/${actorId}`,
+    });
+
+    /*
+    TODO: We need to fetch input schema and prefill input regarding that. But we need to have
```

Comment:
Let's create issues here at Github instead of TODOs

Reply:
#9